### PR TITLE
apps: exim-stats

### PR DESCRIPTION
**Pull request by crcro**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

exim-stats app, including only frozen emails as requested #5008
requires librenms-agent script

### CODE REVIEW

**Comment by laf** [comment]
Thanks, the original issue was badly formatted and it wasn't just frozen :)

I've updated it to show what was meant.

**Comment by crcro** [comment]
@laf updated to include the queue also, thanks!

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -371,7 +372,21 @@ extend ups-apcups /etc/snmp/ups-apcups.sh
 4. Restart snmpd on your host
 5. On the device page in Librenms, edit your host and check the `UPS apcups` under the Applications tab.
 
+### EXIM Stats
+SNMP extend script to get your exim stats data into your host.
 
+##### SNMP Extend
+1. Copy the [exim stats](https://github.com/crcro/librenms-agent/blob/1c091800b713e26395796b5d5379421fc268a39b/snmp/exim-stats.sh) to `/etc/snmp/` (or any other suitable location) on your host.
```
Comment (laf) [inline doc/Extensions/Applications.md]: Needs updating to be the librenms/librenms-agent repo url

Comment (crcro) [inline doc/Extensions/Applications.md]: fixed, thx @laf

**Pull request merged**